zcash_primitives/src/sapling/keys.rs

```diff
@@ -262,6 +280,62 @@ impl Diversifier {
     }
 }
 
+/// The diversified transmission key for a given payment address.
+///
+/// Defined in [Zcash Protocol Spec § 4.2.2: Sapling Key Components][saplingkeycomponents].
+///
+/// Note that this type is allowed to be the identity in the protocol, but we reject this
+/// in [`PaymentAddress::from_parts`].
```

Comment:
```suggestion
/// Note that we reject the identity point, in [`PaymentAddress::from_parts`].
```
It was a mistake to change the protocol to allow this point; see https://github.com/zcash/zcash/issues/3827#issuecomment-1402461041

Reply:
Deferred to #762.